app/src/main/res/xml/preference_payment_setting.xml

```diff
@@ -227,6 +227,11 @@
             app:key="@string/payment_preference_grabpay_key"
             app:title="@string/payment_preference_grabpay_title" />
 
+        <CheckBoxPreference
+                app:defaultValue="true"
+                app:key="@string/payment_preference_atome_key"
+                app:title="@string/payment_preference_atome_title" />
```

Comment:
I think the indentation is a bit longer than the above.

Reply:
Yeah, all indentations are fixed now, thanks!